fury/tests/test_material.py

```diff
@@ -0,0 +1,871 @@
+    material.manifest_pbr(slicer)
+    scene.add(slicer)
+    """
+
+    """
+    # Contour from label setup
+    data = np.zeros((50, 50, 50))
+    data[5:15, 1:10, 25] = 1.
+    data[25:35, 1:10, 25] = 2.
+    data[40:49, 1:10, 25] = 3.
+    color = np.array([[255, 0, 0, 0.6],
+                      [0, 255, 0, 0.5],
+                      [0, 0, 255, 1.0]])
+    surface = actor.contour_from_label(data, color=color)
+    material.manifest_pbr(surface)
+    scene.add(surface)
+    """
+
+    """
+    # Scalar bar setup
+    lut = actor.colormap_lookup_table(
+        scale_range=(0., 100.), hue_range=(0., 0.1), saturation_range=(1, 1),
+        value_range=(1., 1))
+    sb_actor = actor.scalar_bar(lut, ' ')
+    material.manifest_pbr(sb_actor)
+    scene.add(sb_actor)
+    """
+
+    """
+    # Billboard setup
+    centers = np.array([[0, 0, 0], [5, -5, 5], [-7, 7, -7], [10, 10, 10],
+                        [10.5, 11.5, 11.5], [12, -12, -12], [-17, 17, 17],
+                        [-22, -22, 22]])
+    colors = np.array([[1, 1, 0], [0, 0, 0], [1, 0, 1], [0, 0, 1], [1, 1, 1],
+                       [1, 0, 0], [0, 1, 0], [0, 1, 1]])
+    scales = [6, .4, 1.2, 1, .2, .7, 3, 2]
+    """
+    fake_sphere = \
+        """
+        float len = length(point);
+        float radius = 1.;
+        if(len > radius)
+            discard;
+        vec3 normalizedPoint = normalize(vec3(point.xy, sqrt(1. - len)));
+        vec3 direction = normalize(vec3(1., 1., 1.));
+        float df_1 = max(0, dot(direction, normalizedPoint));
+        float sf_1 = pow(df_1, 24);
+        fragOutput0 = vec4(max(df_1 * color, sf_1 * vec3(1)), 1);
+        """
+    """
+    billboard_actor = actor.billboard(centers, colors=colors, scales=scales,
+                                      fs_impl=fake_sphere)
+    material.manifest_pbr(billboard_actor)
+    scene.add(billboard_actor)
+    """
+
+    """
+    # Text3D setup
+    msg = 'I \nlove\n FURY'
+    txt_actor = actor.text_3d(msg)
+    material.manifest_pbr(txt_actor)
+    scene.add(txt_actor)
+    """
+
+    """
+    # Figure setup
+    arr = (255 * np.ones((512, 212, 4))).astype('uint8')
+    arr[20:40, 20:40, 3] = 0
+    tp = actor.figure(arr)
+    material.manifest_pbr(tp)
+    scene.add(tp)
+    """
+
+    if interactive:
+        window.show(scene)
+
+
+def test_manifest_standard(interactive=False):
+    # Setup surface
+    import math
+    import random
+    from scipy.spatial import Delaunay
+    size = 11
+    vertices = list()
+    for i in range(-size, size):
+        for j in range(-size, size):
+            fact1 = - math.sin(i) * math.cos(j)
+            fact2 = - math.exp(abs(1 - math.sqrt(i ** 2 + j ** 2) / math.pi))
+            z_coord = -abs(fact1 * fact2)
+            vertices.append([i, j, z_coord])
+    c_arr = np.random.rand(len(vertices), 3)
+    random.shuffle(vertices)
+    vertices = np.array(vertices)
+    tri = Delaunay(vertices[:, [0, 1]])
+    faces = tri.simplices
```

Comment:
duplicate, maybe you can create a function above named `generate_data` or similar